### PR TITLE
Containers: Replace health check of podman test

### PR DESCRIPTION
**Pull request by ilausuch**
Problem: In recent test for systemd we use inspect health. But in some podman old versions doen't work
Solution: Use a different way to check it

- Related ticket: 
- Verification run: 
- http://openqa.suse.de/tests/12230111
- http://openqa.suse.de/tests/12230140
- http://openqa.suse.de/tests/12230141
- http://openqa.suse.de/tests/12230190 (last small change)

### CODE REVIEW

File: tests/containers/podman_bci_systemd.pm [hunk 1]
```diff
@@ -30,10 +32,8 @@ sub run {
     record_info('Test', 'Start nginx');
     assert_script_run("podman exec nginx systemctl start nginx");
 
-    record_info('Test', 'Wait for container to be healthy');
-    script_retry("podman inspect -f '{{.State.Health.Status}}' nginx | grep -x healthy", retry => 10, delay => 15);
-
-    record_info('Nginx service status', validate_script_output_retry("podman exec nginx systemctl status nginx", qr/running/));
+    record_info('Test', 'Wait for nginx to be healthy');
+    script_retry("podman exec nginx systemctl status nginx | grep Active | grep running", retry => 10, delay => 15);
```
Comment (ricardobranco777) [inline tests/containers/podman_bci_systemd.pm:36]:
```suggestion
    script_retry("podman exec nginx systemctl status -P ActiveState -P SubState nginx | grep -Pz 'active\nrunning'", retry => 10, delay => 15);
```

File: tests/containers/podman_bci_systemd.pm [hunk 2]
```diff
@@ -18,8 +18,10 @@ sub run {
     my $podman = $self->containers_factory('podman');
     $self->{podman} = $podman;
 
+    record_info("Podman version", script_output("podman version"));
+
     record_info('Test', 'Launch a container with systemd');
-    assert_script_run("podman run -d -p 80:80 --health-cmd='curl http://localhost' --name nginx registry.suse.com/bci/bci-init:latest");
+    assert_script_run("podman run -d -p 80:80 --name nginx registry.suse.com/bci/bci-init:latest");
```
Comment (ricardobranco777) [inline tests/containers/podman_bci_systemd.pm:24]: How podman will know that the container is healthy? =/

Comment (ilausuch) [reply tests/containers/podman_bci_systemd.pm:24]: Podman doesn't need to know that in this case, but I can keep it

File: tests/containers/podman_bci_systemd.pm (hunk 1 above)
Comment (ilausuch) [reply tests/containers/podman_bci_systemd.pm]: done

**Review by ricardobranco777 — APPROVED** [review]
LGTM

**Comment by ilausuch** [comment]
It's working

**Pull request merged**